server/config/config.js

```diff
@@ -9,8 +9,7 @@ module.exports = {
     password: process.env.DB_PASSWORD,
     database: process.env.DB_DEV,
     host: process.env.DB_HOST,
-    dialect: process.env.DIALECT,
-    logging: true
+    dialect: process.env.DIALECT
```

Comment:
The default value for the logging parameter displays the log so adding logging: true is redundant and secondly, logging: true is deprecated